tests/openmetrics/test_parser.py

```diff
@@ -723,7 +723,7 @@ def collect(self):
 
         registry = CollectorRegistry()
         registry.register(TextCollector())
-        self.assertEqual(text.encode('utf-8'), generate_latest(registry))
+        self.assertEqual(text.encode('utf-8'), generate_latest(registry, ALLOWUTF8))
```

Comment:
This probably doesn't actually need to change since there is a default value now.